### ISSUE TITLE
User preferences?

### ISSUE THREAD

**Issue by klonuo** [Nevensky/SublimeFortran#22]
Maybe there should be setting preferences, like most ST plugins have?

For example, for setting `gfortran` executable?

On my system `gfortran` is not on PATH. So I either have to put the folder on my PATH (which I don't want to do, as I dont want to expose all programs from `mingw/bin` folder just like that) or manually change lines where `gfortran` is referenced.

**Comment by 315234** [comment]
As far as I know this is not possible in the build system. I've tried using `${fortran_gfortran_executable:gfortran}` as the executable, but only certain variables are allowed. You can already configure the executable and path for the linter using SublimeLinter's settings.

**Comment by klonuo** [comment]
You are right... I should just wrap `gfortran` in batch file and put it on path.

**Comment by 315234** [comment]
I'll put in an API request for this.
